backend/internal/adapter/call_resource.go

```diff
@@ -0,0 +1,45 @@
+package adapter
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/internal/convert"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/internal/resource"
+	"github.com/grafana/grafana-plugin-sdk-go/genproto/pluginv2"
+)
+
+func (a *SDKAdapter) CallResource(ctx context.Context, protoReq *pluginv2.CallResource_Request) (*pluginv2.CallResource_Response, error) {
+	r, exists := a.schema.Resources[protoReq.ResourceName]
+	if !exists {
+		return &pluginv2.CallResource_Response{
+			Code: 404,
```

Comment:
since already imported, maybe `http.StatusNotFound` etc for codes.

Reply:
Will fix :+1: